lib/containers/runtime.pm

```diff
@@ -108,6 +160,24 @@ sub enum_containers {
     return \@containers;
 }
 
+=head2 get_images_by_repo_name
```

Comment:
Why not simply use `get_images`? That's in the end what's happening and `get_images_by_repo_name` suggests, that there are other ways of getting the images as well.

Reply:
`$runtime images` returns the images too. From that you can get an array with the hash id, the name or the repo according to the filter. the function is named to show that the returned array is one of the repo_names. Maybe not the best name but unless you have something better than `get_images` i will skip this suggestion

Reply:
Well, since this is the canonical representation for human beings I still think `get_images` is better, or perhaps `get_images_canonical`. I'm not against leaving it like it is though. I just find it unnecessarily explicit in a place, where there is only one `get_images` method (yet).

Reply:
would `get_repo_names` work for you?

Reply:
I think we should leave it at `get_images_by_repo_name`.